# Generated for NaoQi method ALExtractor.getOutputNames
# Get the list of values updated in ALMemory.

---
# Array of values updated by this extractor in ALMemory
string[] value


